src/net/end_point.h

```diff
@@ -26,6 +30,8 @@ class EndPoint : public std::enable_shared_from_this<EndPoint> {
   int type() const { return SOCK_STREAM; }
   int protocol() const { return protocol_; }
 
+  String Address() const;
```

Comment:
I don't like such interface extension, since the `EndPoint` is abstraction over all possible end-points. Even now unix-socket doesn't have neither host nor port. What do you need it for?